frontend/src/app/modules/projects/project-list/project-list.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="card-container">
+    <mat-card *ngFor="let project of projects">
```

Comment:
I mean component for solo project card, for example project-card or project-details that represent only one project, and at the project page use ngFor to this

Reply:
Fixed